dataland-backend/src/main/kotlin/org/dataland/datalandbackend/frameworks/sme/model/SmeData.kt

```diff
@@ -0,0 +1,25 @@
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
+package org.dataland.datalandbackend.frameworks.sme.model
+
+import jakarta.validation.Valid
+import org.dataland.datalandbackend.annotations.DataType
+import org.dataland.datalandbackend.frameworks.sme.model.general.SmeGeneral
+import org.dataland.datalandbackend.frameworks.sme.model.insurances.SmeInsurances
+import org.dataland.datalandbackend.frameworks.sme.model.power.SmePower
+
+/**
+ * The root data-model for the Sme Framework
+ */
+@Suppress("MagicNumber")
+@DataType("sme", 4)
```

Comment:
Was 5 previously. Was it changed due to the ordering conflict with sfdr? Now it has one with p2p.

Reply:
Ja stimmt, macht keinen Sinn.

Ich hab jetzt die orders von allen frameworks nochmal überarbeitet, und jedes hat jetzt eine eigene order-number.

Ich habe die deutschsprachigen (wie einst von Julia gewünscht) ans Ende der Liste gepackt, und sme einfach direkt davor.
Finde das macht Sinn.